### ISSUE TITLE
[Suggestion] Add synced lyrics view similar to Spotify's lyrics

### ISSUE THREAD

**Issue by tqqk** [TecCheck/FastLyrics#32]
Hey, 
I love this project, it even replaced the official Genius app for me. I was wondering if it would be possible to add lyrics view similar to Spotify's (bigger text, few lines) to make following the lyrics easier. Another user mentioned an option of preferred lyrics mode, which is probably already on your radar, but it would be amazing to combine it with a new option for a lyrics view.

**Comment by TecCheck** [comment]
Yes, both of these are on my radar. However I've never done any custom views so this is quite difficult for me and will probably take some time. I hope this is fine and thanks for using my app :D

**Comment by TecCheck** [comment]
I've found a somewhat good library for it and opened #33. If you want you can check it out. I'll make a test build soon

**Comment by teccheck** [comment]
This is implemented now. Hope you like it
